en/getting_started/sensor_selection.md

```diff
@@ -1,61 +1,60 @@
+
+Options include the original 3DR GPS, LEA 6H, NEO 7M, NEO 8 GPS+Compass solutions.
 
 ![3DR GPS_Top](../../images/gps_3dr_top_1.jpg) 
 ![GPS + Compass](../../images/gps_compass.jpg)
 
-Because the internal compass of Pixhawk is close to other electronics it
-could be susceptible to electromagnetic noise. So when we use the
-external Compass we can locate it away from electronics on a pedestal or
-at the wing of the plane.
 
 ## Airspeed
 
-The Airspeed sensor is mandatory for most of the Airplane Frames and for
-all VTOL Frames. It is very important sensor because the PX4 autopilot
-does not have other means to detect stall, except with the airspeed
-sensor. For the airplane it is the airspeed that guarantees lift not
-ground speed. We recommend using the digital airspeed sensor.
+The Airspeed sensor is required for most of fixed-wing frames and for all VTOL frames.
+
+This sensor is so important because the autopilot does not have other means to detect stall. 
+For fixed-wing vehicles it is the airspeed that guarantees lift not ground speed!
+
+We recommend using a digital airspeed sensor.
 
 ![Digital airspeed sensor](../../images/digital_airspeed_sensor.jpg)
 
 ## Distance
 
-There are more and more affordable Lidar Rangefinder options. These
-sensors are used during the final phase of landing for precise altitude
-measuring. They are also used for avoidance. PulsedLight launched their
-Lidar-Lite2 Laser Rangefinder. as an economical laser based measurement
-solution supporting a wide variety of applications like in drones. It
-measures distance, velocity and signal strength of cooperative and non
-cooperative targets at distances from zero to more than 40 meters.
-Offering the highest performance available in a single beam ranging
-sensor in its class.
+Distance sensors are used for precision landing, object avoidance and terrain following.
+
+PX4 supports many affordable distance sensors, using different technologies, and supporting different ranges and features. For more information see: [Rangerfinders](../sensor/rangefinders.md).
 
-![lidar_lite_1](../../images/lidar_lite_1.png)
+<img src="../../images/lidar_lite_1.png" title="lidar_lite_1" width="500px" />
 
-## Position
+## Optical Flow
 
-Flow is an optical flow smart camera that can track motion, as well has
-as integrated sonar sensor. The blended information is fed to the
-Pixhawk to have one of the most locked in position holds available. It
-is used as a Positioning sensor when there is no GPS signal like
-indoors.
+Flow is an optical flow smart camera that can track motion, and has as integrated sonar sensor.
+PX4 blends the sensor output with information from other position sources (e.g. GPS) to provide a more accurate position lock. 
+This sensor can be used indoors, when no GPS signal is available.
 
 ![px4flow-bottom](../../images/px4flow_bottom.jpg)
 
-For the wiring of all sensors see: [Sensors Wiring](TBD).
 
+## Sensor Wiring {#wiring}

```

Comment:
Our wiring story for sensors is all over the place - like our docs on them. Here I've just tried to provide links to where more information about sensor hardware can be found.